app/helpers/concerns/foreman_bootdisk/hosts_helper_ext.rb

```diff
@@ -19,6 +20,15 @@ def host_title_actions_with_bootdisk(*args)
     host_title_actions_without_bootdisk(*args)
   end
 
+  # need to wrap this one in a test for template proxy presence
+  def display_bootdisk_for_subnet
+    if (proxy = @host.try(:subnet).try(:tftp)) && proxy.features.map(&:name).include?('Templates')
```

Comment:
I don't think it is worth it, we actually do not use it in this form anywhere else. But improving this to:

        if (proxy = @host.try(:subnet).try(:tftp)) && proxy.has_feature?('Templates')